app/scripts/views/mixins/session-verification-poll-mixin.js

```diff
@@ -96,7 +96,7 @@ define(function (require, exports, module) {
         // if the user is switching networks.
         // Report a known error to Sentry, but not the user.
         // Details: github.com/mozilla/fxa-content-server/issues/2638.
-        this.logError(AuthErrors.toError('POLLING_FAILED'));
+        this.logError(AuthErrors.toError('POLLING_FAILED', err));
```

Comment:
Ah, for some reason I just assumed it'd be stringified into JSON. Instead, I've opted to call `JSON.stringify(err)` here.

The error that comes back from fxa-js-client *might* have a status on it, but not always. If there is a network error, such that `xhr.status === 0`, then the error `{ error: "Timeout error", errno: 999 }` is returned. I'd want to be able to see that it was actually a network error, and so the only thing that would identify this is the `error` property of `"Timeout error"`.